idea/testData/inspectionsLocal/convertToObjectDeclaration/refactorCompanionReference.kt.after

```diff
@@ -0,0 +1,11 @@
+// FIX: Convert to object declaration
+// WITH_RUNTIME
+
+object A {
+    val prop = "test"
+}
+
+fun main() {
```

Comment:
Test fails, please fix, else it is good!

Reply:
Rebased to current master, was a bug we saw before.